deep_qa/common/checks.py

```diff
@@ -30,9 +30,5 @@ def log_keras_version_info():
         backend = K.backend()
     except AttributeError:
         backend = K._BACKEND  # pylint: disable=protected-access
-    if backend == 'theano':
-        import theano
-        logger.info("Theano version: " + theano.__version__)
-    elif backend == 'tensorflow':
-        import tensorflow
-        logger.info("Tensorflow version: " + tensorflow.__version__)  # pylint: disable=no-member
+    import tensorflow
```

Comment:
perhaps it'd be better to move all of these `import tensorflow` statements (there are a few scattered around the diff) to the top of the file, since we're unconditionally using it anyway?